Cargo.toml

```diff
@@ -20,10 +25,11 @@ panic = 'abort'
 rpath = false
 
 [workspace.dependencies]
-polytone = { path = "packages/polytone", version = "1.0.0" }
-polytone-proxy = { path = "contracts/main/proxy", version = "1.0.0" }
-polytone-note = { path = "contracts/main/note", version = "1.0.0" }
-polytone-voice = { path = "contracts/main/voice", version = "1.0.0" }
+polytone = { package = "abstract-polytone", path = "packages/polytone", version = "1.0.0" }
+polytone-proxy = { package = "abstract-polytone-proxy", path = "contracts/main/proxy", version = "3.0.0" }
+polytone-note = { package = "abstract-polytone-note", path = "contracts/main/note", version = "3.0.0" }
+polytone-voice = { package = "abstract-polytone-voice", path = "contracts/main/voice", version = "3.0.0" }
+cw-orch-polytone = { package = "abstract-cw-orch-polytone", path = "packages/cw-orch-polytone", version = "4.0.1" }
```

Comment:
I don't like the idea of major bumping because of changes of cw-orch, should we do minor bump instead?

Reply:
We can do major bump like discussed.